client/ayon_core/pipeline/delivery.py

```diff
@@ -409,5 +409,9 @@ def get_representations_delivery_template_data(
             "version": version_entity["version"],
         })
         _merge_data(template_data, repre_entity["context"])
+
+        # Remove roots from temlate data to auto-fill them with anatomy data
+        template_data.pop("root")
```

Comment:
Could it be - for whatever reason.... that `root` does _not_ exist in context? If so, this may need to be `template_data.pop("root", None)` because otherwise [it'd raise an `IndexError`](https://docs.python.org/3/tutorial/datastructures.html#more-on-lists)

Reply:
It should be, and I actually had it, now I'm a little bit confused, because I saw it in code.
```suggestion
        template_data.pop("root", none)
```